<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>areaconta</class>
 <widget class="QDialog" name="areaconta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>780</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>940</width>
    <height>780</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>940</width>
    <height>780</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color:white;
	color:white;
}</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>10</width>
     <height>781</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>10</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>10</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">Line{
	background-color:rgb(59,81,127);
	border:none;
}</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border:1px solid white;
	background-image: url(:/area/area/saida2.png);
	background-repeat:none;
	font: 10pt &quot;Segoe UI&quot;;
	background-color: rgb(72, 145, 217);
	color:white;
}
QPushButton:hover{
	border:2px solid #0090d8;
}</string>
   </property>
   <property name="text">
    <string>VOLTAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 900 15pt &quot;Segoe UI&quot;;
	color: rgb(255, 212, 38);
}</string>
   </property>
   <property name="text">
    <string>CASH</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>41</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 600 45pt &quot;Segoe UI Semibold&quot;;
	color: rgb(185, 185, 185);
}
</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 13pt &quot;Segoe UI &quot;;
	color: rgb(59,81,127);
}
</string>
   </property>
   <property name="text">
    <string>VER CONTA(S) CAMBA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>190</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-image: url(:/area/area/relatorio.png);
	background-repeat:none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>180</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>30</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit{
	border:1px solid rgb(59, 81, 127);
	background-color: white;
	color:rgb(59, 81, 127);
	font: 10pt &quot;Segoe UI&quot;;
}

</string>
   </property>
   <property name="locale">
    <locale language="Portuguese" country="Portugal"/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="timeSpec">
    <enum>Qt::UTC</enum>
   </property>
   <property name="date">
    <date>
     <year>2023</year>
     <month>11</month>
     <day>23</day>
    </date>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>180</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border:1px solid white;
	image: url(:/area/area/cash.png);
	background-repeat:none;
	background-color: rgb(72, 145, 217);
}
QPushButton:hover{
	border:2px solid #0090d8;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>649</width>
     <height>351</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>649</width>
     <height>351</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>649</width>
     <height>351</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget{
	background-color: rgb(59,81,127);
	color:white;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::DashLine</enum>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>36</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>106</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>36</number>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Contaº</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Área</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Dinheiro-IN (AOA)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Data</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Hora</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Total (AOA)</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_edit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>640</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border:1px solid white;
	background-repeat:none;
	font: 10pt &quot;Segoe UI&quot;;
	background-color: rgb(72, 145, 217);
	color:white;
}
QPushButton:hover{
	border:2px solid #0090d8;
}</string>
   </property>
   <property name="text">
    <string>EDITAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>710</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border:1px solid white;
	background-repeat:none;
	font: 10pt &quot;Segoe UI&quot;;
	background-color: rgb(72, 145, 217);
	color:white;
}
QPushButton:hover{
	border:2px solid #0090d8;
}</string>
   </property>
   <property name="text">
    <string>DELETAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_relatory_pdf">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>600</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border:1px solid white;
	background-repeat:none;
	font: 10pt &quot;Segoe UI&quot;;
	background-color:rgb(59,81,127);
	color:white;
}
QPushButton:hover{
	border:1px solid #0090d8;
}</string>
   </property>
   <property name="text">
    <string>RELATÓRIO</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_Delete">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>710</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit{
	border:1px solid rgb(59, 81, 127);
	background-color:white;
	color:rgb(59, 81, 127);
	font: 10pt &quot;Segoe UI&quot;;
}

</string>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="timeSpec">
    <enum>Qt::UTC</enum>
   </property>
   <property name="date">
    <date>
     <year>2023</year>
     <month>11</month>
     <day>23</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>680</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 9pt &quot;Segoe UI &quot;;
	color:rgb(59,81,127);
}
</string>
   </property>
   <property name="text">
    <string>SELECIONE DATA E D/E N* CONTAS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>610</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 9pt &quot;Segoe UI &quot;;
	color:rgb(59,81,127);
}
</string>
   </property>
   <property name="text">
    <string> CONTA E/  N^1 CONTA</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_month">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>180</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>30</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
	background-color:white;
	border:1px solid rgb(59, 81, 127);
	color: rgb(59, 81, 127);
}</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Janeiro</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fevereiro</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Março</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Abril</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Maio</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Junho</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Julho</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Agosto</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Setembro</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Outubro</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Novembro</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dezembro</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>160</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 9pt &quot;Segoe UI &quot;;
	color: rgb(59,81,127);
}
</string>
   </property>
   <property name="text">
    <string>MÊS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>160</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 9pt &quot;Segoe UI &quot;;
	color: rgb(59,81,127);
}
</string>
   </property>
   <property name="text">
    <string>DATA</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>dateEdit</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
